### ISSUE TITLE
Unable to connect to the website using IPv6

### ISSUE THREAD

**Issue by malemburg** [EuroPython/epcon#1372]
See https://www.ssllabs.com/ssltest/analyze.html?d=ep2021.europython.eu&latest

We do have IPv6 configured on the server, so it's not clear why this isn't working.

**Comment by fpierfed** [comment]
Good catch! Seems to be related to docker-compose and ipv6. See fir instance these blog posts and comments therein: https://dev.to/csgeek/docker-ipv6-guide-235d

**Comment by fpierfed** [comment]
I have deleted AAAA DNS records for epstage and ep2021. Not a fix, or course but we can postpone this issue for now.
